transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/DataTypeSerializationHelper.groovy

```diff
@@ -0,0 +1,31 @@
+package org.transmartproject.rest.marshallers
+
+import org.transmartproject.export.Datatypes
+
+class DataTypeSerializationHelper  extends AbstractHalOrJsonSerializationHelper<Datatypes>{
+
+    final Class targetType = Datatypes
+    final String collectionName = 'datatypes'
+
+    @Override
+    Map<String, Object> convertToMap(Datatypes datatypes) {
+        def cohortInfoList = []
+        def cohortsMap = [:]
+        datatypes.OntologyTermsMap.each { ID, terms ->
```

Comment:
Please stick to the convention that:
- Class names start with capitals
- Constants are all caps,
- Variables, fields, method names start with lower case.